src/Sulu/Bundle/MediaBundle/Entity/MediaDataProviderRepository.php

```diff
@@ -74,7 +90,17 @@ public function findByFilters($filters, $page, $pageSize, $limit, $locale, $opti
             $filters['dataSource'] = null;
         }
 
-        $entities = $this->parentFindByFilters($filters, $page, $pageSize, $limit, $locale, $options);
+        $entities = $this->parentFindByFilters(
+            $filters,
+            $page,
+            $pageSize,
+            $limit,
+            $locale,
+            $options,
+            $user,
+            Collection::class,
+            'collection'
```

Comment:
In order to activate permissions for other entities, they also have to override the default `findByFilters` method in this way, in order to pass the last two arguments. Is that good enough?